File: back/src/main/java/org/bliz/trivial/controller/model/QuestionsPerGamePK.java
package org.bliz.trivial.controller.model;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class QuestionsPerGamePK implements Serializable {

	private static final long serialVersionUID = -4346754443111685999L;

	private Integer questionCode;

	private Integer gameCode;

	private Integer userCode;
}

File: front/src/app/models/questionsPerGamePK.ts
export class QuestionsPerGamePK {

    questionCode: number;
    gameCode: number;
    userCode: number;

    constructor(questionCode: number, gameCode: number, userCode: number) {
        this.questionCode = questionCode;
        this.gameCode = gameCode;
        this.userCode = userCode;
    }
}